packages/merkle-distributor/src/IpfsHelper.ts

```diff
@@ -0,0 +1,44 @@
+// This script adds some useful functions for dealing with IPFS.
+
+const ipfsApi = require("ipfs-http-client");
+const nodeFetch = require("node-fetch");
```

Comment:
im not sure if this will solve all our fe problems, but there is an [isomorphic-fetch](https://www.npmjs.com/package/isomorphic-fetch) which basically switches between node fetch or fetch based on environment detected